src/colvar.h

```diff
@@ -655,6 +655,10 @@ inline colvarvalue const & colvar::total_force() const
 
 inline void colvar::add_bias_force(colvarvalue const &force)
 {
+  if (! is_enabled(f_cv_gradient)) {
```

Comment:
In principle, this kind of sanity checks would not be necessary at run-time in production. I wonder if it could be a good idea to have a build flag enabling these tests, and allowing us to differentiate test builds and production builds? Or it could be combined with debug mode for simplicity...

For now though, I think this is the way to go.

Reply:
This issue was due to an uncommon scenario: a class written before `colvardeps` was introduced, but integrated during the transition period.  A test would have caught it, unfortunately (it will now).

This particular check won't add much overhead, but it's a good suggestion to have the ability to turn them off as needed.  Replacing it with an inline function within `colvardeps`.